backend/alembic/versions/3c5e35aa9af0_polling_document_count.py

```diff
@@ -35,6 +35,7 @@ def upgrade() -> None:
                 "SUCCESS",
                 "FAILED",
                 name="indexingstatus",
```

Comment:
can we add comments for all these so we know that these were edited after the fact just in case we run into issues